build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -60,15 +62,25 @@
                             SetupRestoreTestsInputs;"
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
-    <Exec Command="$(DotnetUnderTest) restore $(TestDirectory) --fallbacksource $(TestPackagesDir)" WorkingDirectory="$(TestDirectory)" />
+    <DotNetRestore  FallbackSource="$(TestPackagesDir)"
+                    ToolPath="$(DotnetUnderTest)" 
+                    WorkingDirectory="$(TestDirectory)" />
   </Target>
  
   <Target Name="CleanTestProjectsBinObj" 
           DependsOnTargets="SetupRestoreTestsInputs;" 
           Inputs="@(RestoreTestsInputs)"
           Outputs="@(RestoreTestsInputs->'%(RelativeDir)project.lock.json')">
-    <Delete Files="$(TestDirectory)/**/bin/**/*.*')" />
-    <Delete Files="$(TestDirectory)/**/obj/**/*.*')" />
+    <ItemGroup>
+      <FilesToDelete Include="$(TestDirectory)**/bin/**/*.*" />
```

Comment:
will probably be faster to delete directories

https://msdn.microsoft.com/en-us/library/xyfz6ddb.aspx